featuretools/entityset/entity.py

```diff
@@ -249,7 +250,7 @@ def query_by_values(self, instance_vals, variable_id=None, columns=None,
             df = self.df.head(0)
 
         else:
-            if isinstance(instance_vals, dd.Series):
+            if isinstance(instance_vals, dd.Series) or isinstance(instance_vals, ks.Series):
```

Comment:
if you prefer it, you can also just just do `isinstance(instance_vals, (ks.Series, dd.Series))`. I find it a bit more readable

Reply:
i actually see you do that form below. anyways, just a nitpik